spec/v1.0/resource_descriptor.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package github.intoto.attestation.spec.v1_0;
```

Comment:
Q: From a package name standpoint, I've been wondering if we would want to simplify it a bit to something like `intoto.attestation.v1_0`? This way, folks could import `intoto.attestation.v1_0.Statement` and `intoto.attestation.v1_0.predicates.VerificationSummary` directly. Seems slightly more intuitive... Just a question, not sure if this would break any proto conventions.

Reply:
That makes sense but it is a bigger change (since we'd want to change the other protos too).  Perhaps it could be a separate PR?

Reply:
Yes, doing this in a separate PR makes sense.

Reply:
I've raised this point before but I'm a little iffy as we don't control intoto.io. Is that a realistic concern?

Reply:
We (well the CNCF tbf) control in-toto.io, but not intoto.io. The hyphen's been dropped in the dsse envelope, our java package and so on. I'm very anti-hyphen for all new projects as a result. If we think it's not an issue (and it hasn't been for the other instances so far I guess), we can go ahead. But https://intoto.io is an actual tech company so...

I registered intoto.link a while ago so we'd have one without a hyphen (and because the pun was great). I've also been trying to find out if intoto.dev was registered by someone we know. Perhaps we can continue with io.intoto until we know more? I'll defer to the rest of you, I just wanted to note it.

Reply:
> But https://intoto.io/ is an actual tech company so...

Oh yikes, maybe I should put it back the way it was?  WDYT @marcelamelara ?

Reply:
Nope, it's taken.

Reply:
Maybe we should use `_` ? https://stackoverflow.com/questions/63225957/how-to-allow-minus-sign-for-package-name-in-proto-file

Reply:
Oh ha... good to know! I'd be fine with using an underscore. Would `in_toto.attestation.` work as a package prefix for the protos then? `in_toto` is also the Python package name I believe...

Reply:
I'm fine with `_` if that's not expressly disallowed!

Reply:
Done.  It is supported and everything seems to work.